WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -112,6 +112,16 @@ final class OrdersRootViewController: UIViewController {
         }
     }
 
+    override var shouldShowOfflineBanner: Bool {
+        // Should show the offline banner only when there's no orderDetailsViewController in memory
+        // otherwise, it will be shown within the order details view, so there's no need to duplicate it
+        if let orderDetailsViewController {
+            return false
+        } else {
+            return true
+        }
+    }
```

Comment:
This bit feels a bit hacky, happy to get any feedback: 

I found very tricky to know from the `OrdersRootViewController` when are we on a split view or not. By checking the different splitViewController, viewController, navigationControllers, etc, ...references, the only difference I could find in this point about using a split view was that the `orderDetailsViewController` will be nil or not.